analyses/oncoprint-landscape/04-alteration-counts-by-cancer-group.R

```diff
@@ -37,7 +37,6 @@ module_dir <- file.path(root_dir, "analyses", "oncoprint-landscape")
 # Source the custom functions script
 source(
   file.path(
-    root_dir,
```

Comment:
Note this was a change I implemented. The module didn't run otherwise; `module_dir` contains `root_dir` so paths were borked.